src/BoxWhisker.js

```diff
@@ -24,6 +24,22 @@ export default class BoxWhisker extends Plot {
     this._discrete = "x";
     this._shape = constant("Box");
     this.x("x");
+    this._shapeConfig = assign(this._shapeConfig, {
+      Box: {
```

Comment:
This shapeConfig should be moved directly into the constructor of `Plot.js`, just like the custom shapeConfig for the other shapes like Area and Bar.